doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -52,6 +52,18 @@ This is a much faster way than waiting for the subscriber node to notice that th
 
 To enable persistent subscriptions, set the :kconfig:option:`CONFIG_CHIP_PERSISTENT_SUBSCRIPTIONS` Kconfig option.
 
+The time that it takes to re-establish all subscriptions and CASE sessions depends on the number of sessions that can be recovered simultaneously.
+If the number of sessions and subscriptions to recover is bigger than the maximum capability, the device will re-establish as many as possible and schedule re-establishment of the rest after specified time interval.
+The wait interval between the following re-establishment attempts impacts the overall time to recover all device connections as well, so decreasing it can be beneficial for the user experience.
+However, the small interval value will impact the device power consumption.
```

Comment:
```suggestion
However, a small interval value will increase device power consumption.
```
I presume this is correct since we started off with "however".

Reply:
Yes